icons/rotate-ccw.json

```diff
@@ -6,7 +6,9 @@
     "ericfennis",
     "danielbayley",
     "jguddas",
-    "karsa-mistmere"
+    "karsa-mistmere",
+    "chessurisme",
+    "realguse"
```

Comment:
Only contributors to the design of the icons should be placed here.

Reply:
well fixed that